README.md

```diff
@@ -64,14 +64,15 @@ docker run --network host -it --rm \
   ghcr.io/gazebosim/ionic_demo_rmf_dashboard:main
 ```
 
-Start the API server docker image, make sure to note the `ROS_DOMAIN_ID` and `RMW_IMPLEMENTATION`. This starts the API server on `localhost:8000`.
+Start the API server docker image, make sure to note the `ROS_DOMAIN_ID` and `RMW_IMPLEMENTATION`. This starts the API server on `localhost:8000`. Example values: `ROS_DOMAIN_ID=1` and `RMW_IMPLEMENTATION=rmw_fastrtps_cpp`
```

Comment:
Oh that'll be useful for users for sure, thanks for adding that! Perhaps we should also provide the links,
* https://docs.ros.org/en/rolling/Concepts/Intermediate/About-Domain-ID.html
* https://docs.ros.org/en/rolling/How-To-Guides/Working-with-multiple-RMW-implementations.html

Reply:
yep good idea. Added. 9c733049b3704b885c7daa3d856f68e4aa672f6f